src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -109,18 +109,29 @@ public IWatcher watch(String namespace, IOpenShiftWatchListener listener, String
 	}
 
 	@Override
+	public IWatcher watch(IOpenShiftWatchListener listener, String...kinds) {
+		return this.watch("", listener, kinds);
+	}
+
+	@Override
 	public String getResourceURI(IResource resource) {
 		return new URLBuilder(getBaseURL(), typeMapper, resource).build().toString();
 	}
 
 	@Override
 	public <T extends IResource> List<T> list(String kind) {
-		return list(kind,""); //assumes namespace=default
```

Comment:
Does this means that we need to do a major release? If the previous behavior here was that empty namespace meant default.